assets/charts/control-plane/kubelet/values.yaml

```diff
@@ -3,3 +3,4 @@ clusterDNS: 10.0.0.10
 clusterDomain: cluster.local
 enableTLSBootstrap: true
 cloudProvider:
+kubernetesCACert: ""
```

Comment:
>What does this mean?

It's mostly for discovery, that such variable is used in the templates. In theory it could be skipped.

 >Do we need to have some default value?

No, since each cluster will have different certificates.

>Can't we set it to something obviously wrong like: this-line-should-have-been-replaced-with-your-ca-cert

We could have a template rule validating that this field is populated, however, we don't do it right now as a general practice, so I think it should be fine :tm: 

Example of forcing render error when value is not populated: https://github.com/flexkube/kube-apiserver-helm-chart/blob/a1b291625c52035977448d26e6e0261b15592650/templates/secret.yaml#L7-L18

Reply:
Added a `required` just to be on the safe side